tools/sync/opengrok.py

```diff
@@ -168,3 +179,22 @@ def delete_project(logger, project, uri):
 
 def get_uri(*uri_parts):
     return '/'.join(s.strip('/') for s in uri_parts)
+
+
+def is_localhost_url(url):
+    """
+    Check if given URL is based on localhost.
+    """
+
+    o = urlparse(url)
+    return o.hostname == 'localhost'
```

Comment:
Maybe add loopback addresses? Or no need for that?

Reply:
can do.